.github/workflows/validation.yaml

```diff
@@ -24,27 +24,27 @@ jobs:
           echo "::set-output name=num-yaml-files::$(find ${{ matrix.release }} -name "*.y*ml" -printf '.' | wc -m | xargs)"
       - name: lint ${{ matrix.release }} yaml
         if: ${{ steps.has-yaml-files.outputs.num-yaml-files != '0' }}
-        uses: ibiqlik/action-yamllint@v1
+        uses: ibiqlik/action-yamllint@v3.1.1
         with:
           file_or_dir: ${{ matrix.release }}/ok*/*.y*ml
           # can match with the following: ^(.*):(\d+):(\d+):\s\[(.*)\]\s(.*)$
           # for status check annotations in GitHub
           format: parsable
-      - uses: jitterbit/get-changed-files@v1
-        name: determine file changes
-        if: ${{ github.event_name == 'pull_request' }}
-        id: changed-files
-        with:
-          format: space-delimited
-          token: ${{ secrets.GITHUB_TOKEN }}
-      - name: open knowledge framework validation (pull request)
-        uses: helpfulengineering/ok-validate@v0.1.0
-        if: ${{ github.event_name == 'pull_request' && steps.has-yaml-files.outputs.num-yaml-files != '0' }}
-        with:
-          file-restrictions: ${{ steps.changed-files.outputs.all }}
-          path: ${{ matrix.release }}/
-      - name: open knowledge framework validation
-        uses: helpfulengineering/ok-validate@v0.1.0
-        if: ${{ github.event_name != 'pull_request' && steps.has-yaml-files.outputs.num-yaml-files != '0' }}
-        with:
-          path: ${{ matrix.release }}/
+      # - uses: jitterbit/get-changed-files@v1
+      #   name: determine file changes
+      #   if: ${{ github.event_name == 'pull_request' }}
+      #   id: changed-files
+      #   with:
+      #     format: space-delimited
+      #     token: ${{ secrets.GITHUB_TOKEN }}
+      # - name: open knowledge framework validation (pull request)
+      #   uses: helpfulengineering/ok-validate@v0.1.0
+      #   if: ${{ github.event_name == 'pull_request' && steps.has-yaml-files.outputs.num-yaml-files != '0' }}
+      #   with:
+      #     file-restrictions: ${{ steps.changed-files.outputs.all }}
+      #     path: ${{ matrix.release }}/
+      # - name: open knowledge framework validation
+      #   uses: helpfulengineering/ok-validate@v0.1.0
+      #   if: ${{ github.event_name != 'pull_request' && steps.has-yaml-files.outputs.num-yaml-files != '0' }}
+      #   with:
+      #     path: ${{ matrix.release }}/
```

Comment:
Any specific reason this is being disabled? It has been a while since I touched this but these sections were responsible for the following:
* Identify all files that were _changed_ as part of the PR
  * This was important because we are trying to fail the pipeline due to errors pertinent to the Pull Request -- if we did not restrict a pipeline might fail due to previously committed files
* Running `ok-validate` against YAML files that were changed in the PR using the restrictions defined above
  * This ensures that the YAML files are valid and adhere to schema standards defined in `ok-validate`

Reply:
Okay, so you are creating alternate tooling for validation as opposed to forking the existing validation (e.g., forking the `okh.yaml` in the [schema dir](https://github.com/helpfulengineering/ok-validate/tree/main/okv/schemas) with `okh-helpful.yaml` etc)?
I only ask because I remember the problem this was trying to solve was having incomplete manifests being committed to the repo. I see several PRs that were merged which had failing pipelines which caught YAML lint and incomplete manifests but were ignored which is why you are seeing failures on your PR.